### PR TITLE
Add readmes

### PR DESCRIPTION
**Pull request by Juan-M-V**
Add more information onto the specific commands needed to run most of the examples.

Depends on #15, #14 and #11

### CODE REVIEW

File: asn1/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Fuzzing example with honggfuzz for the asn1 crate.
+
+From the [asn1](https://crates.io/crates/asn1) readme:
+> This is a Rust library for parsing and generating ASN.1 data (DER only).
+
+
+## How to run
+
+Simply run the command `cargo hfuzz run asn1`
```
Comment (daphneherlambda) [inline asn1/README.md:11]: you can mention in this section of the readmes in which folder you have to be in to run the command, in this case the folder is **asn1** but for the ones made with cargofuzz its the fuzz folder within the fuzzer

File: wasmbin_fuzz/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Fuzzing example with `cargo-fuzz` for the [wasmbin](https://crates.io/crates/wasmbin) crate
+
+From the library's readme:
+> wasmbin is a library implementing parsing and serialization WebAssembly binaries.
+
+## How to run
+
+Run any of the targets with: `cargo +nightly fuzz run decode`
+The available targets are `decode`, `encode` and `roundtrip`
```
Comment (daphneherlambda) [inline wasmbin_fuzz/README.md]: the roundtrip one was deleted so don´t mention it

**Review by daphneherlambda — CHANGES_REQUESTED** [review]

**Comment by daphneherlambda** [comment]
Very necesary Readmes, just a few details and the readme for the bencode_parser and cairo_fuzz are missing

File: asn1/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+### Asn1 fuzzer
+
+Fuzzing example with honggfuzz for the asn1 crate.
+
+From the [asn1](https://crates.io/crates/asn1) readme:
```
Comment (Loco1982) [inline asn1/README.md]:
```suggestion
From the [asn1](https://crates.io/crates/asn1) README:
```

File: deflate_fuzz/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+### Deflate fuzzer
+
+Diferential fuzzing example for the crates: [libflate](https://crates.io/crates/libflate) and [inflate](https://crates.io/crates/inflate) using `cargo-fuzz`. Both libraries decode data compressed with the [DEFLATE](https://en.wikipedia.org/wiki/Deflate) algorithm, libflate also allows for encoding.
```
Comment (Loco1982) [inline deflate_fuzz/README.md]:
```suggestion
Differential fuzzing example for the crates: [libflate](https://crates.io/crates/libflate) and [inflate](https://crates.io/crates/inflate) using `cargo-fuzz`. Both libraries decode data compressed with the [DEFLATE](https://en.wikipedia.org/wiki/Deflate) algorithm, libflate also allows for encoding.
```

File: image-tiff/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+### Tiff fuzzer
+
+Fuzzing example with honggfuzz for the tiff crate.
+
+From the [tiff](https://crates.io/crates/tiff) readme:
```
Comment (Loco1982) [inline image-tiff/README.md]:
```suggestion
From the [tiff](https://crates.io/crates/tiff) README:
```

File: lambdaworks_fuzz/README.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+### Lambdaworks fuzzer
+
+Fuzzing example for the [lambdaworks](https://github.com/lambdaclass/lambdaworks) project using `cargo-fuzz`.
+
+From the project's readme:
```
Comment (Loco1982) [inline lambdaworks_fuzz/README.md]:
```suggestion
From the project's README:
```

File: merkle_patricia_tree/README.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+### Merkle patricia tree fuzzer
+
+Fuzzing example with honggfuzz for the Patricia Merkle Tree project.
+
+From the [`merke_patricia_trie`](https://github.com/lambdaclass/merkle_patricia_tree) readme:
```
Comment (Loco1982) [inline merkle_patricia_tree/README.md]:
```suggestion
From the [`merke_patricia_tree`](https://github.com/lambdaclass/merkle_patricia_tree) README:
```

File: merkle_patricia_tree/README.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+> Implementation of Ethereum's Patricia Merkle tree in Rust
```
Comment (Loco1982) [inline merkle_patricia_tree/README.md]:
```suggestion
> Implementation of Ethereum's Patricia Merkle Tree in Rust
```

File: wasmbin_fuzz/README.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+### Wasmbin fuzzer
+
+Fuzzing example with `cargo-fuzz` for the [wasmbin](https://crates.io/crates/wasmbin) crate
+
+From the library's readme:
```
Comment (Loco1982) [inline wasmbin_fuzz/README.md]:
```suggestion
From the library's README:
```

File: xmlparser/README.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
+### Xmlparser fuzzer
+
+Fuzzing example with honggfuzz for the xmlparser crate.
+
+From the [xmlparser](https://crates.io/crates/xmlparser) readme:
```
Comment (Loco1982) [inline xmlparser/README.md]:
```suggestion
From the [xmlparser](https://crates.io/crates/xmlparser) README:
```

**Review by Loco1982 — CHANGES_REQUESTED** [review]
Minor spelling stuff

File: wasmbin_fuzz/README.md (hunk 2 above)
Comment (Juan-M-V) [reply wasmbin_fuzz/README.md]: Resolved: https://github.com/lambdaclass/fuzzing_examples/pull/16/commits/30deb212e89da93e4580217e917f3704eac4a5c4

File: asn1/README.md (hunk 1 above)
Comment (Juan-M-V) [reply asn1/README.md:11]: Resolved: https://github.com/lambdaclass/fuzzing_examples/pull/16/commits/30deb212e89da93e4580217e917f3704eac4a5c4

File: cairo_fuzz/README.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
+### Asn1 fuzzer
+
+Fuzzing example with honggfuzz for the cairo-rs project.
+
+From the [cairo-rs](https://github.com/lambdaclass/cairo-rs) readme:
```
Comment (Loco1982) [inline cairo_fuzz/README.md]:
```suggestion
From the [cairo-rs](https://github.com/lambdaclass/cairo-rs) README:
```

**Review by Loco1982 — APPROVED** [review]
Minor detail

File: cairo_fuzz/fuzz/Cargo.lock [hunk 12]
```diff
@@ -136,10 +136,24 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "572f695136211188308f16ad2ca5c851a712c464060ae6974944458eb83880ba"
 
 [[package]]
-name = "cairo-rs"
-version = "0.1.0"
+name = "cairo-felt"
```
Comment (daphneherlambda) [inline cairo_fuzz/fuzz/Cargo.lock]: undo the changes in this file, as it doesn´t belong to the readmes

**Review by daphneherlambda — CHANGES_REQUESTED** [review]

**Review by daphneherlambda — APPROVED** [review]
